src/main.js

```diff
@@ -18,7 +18,7 @@ var nomnom = require("nomnom")
     .option("output", {
         position: 1,
         metavar: "FILE",
-        help: "Write JavaScript to this file",
+        help: "Write JavaScript to this file, if omitted then run the input",
```

Comment:
This is kinda confusing. I think if Squiggle can run code instead of just compiling it, this should be moved into like a `-o FILE` option which can be omitted.